### PR TITLE
Geometries: Clean up TS files.

### PR DESCRIPTION
**Pull request by klevron**
**Description**

Fix a little typo in param comment for `thetaLength`

### CODE REVIEW

**Comment by Mugen87** [comment]
Can you please also fix this for `CylinderGeometry.d.ts`?

**Comment by klevron** [comment]
Ok, done, and added a fix for Cone

File: src/geometries/CylinderBufferGeometry.d.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export class CylinderBufferGeometry extends BufferGeometry {
 	 * @param [heightSegments=1] — Number of rows of faces along the height of the cylinder.
```
Comment (Mugen87) [inline src/geometries/CylinderBufferGeometry.d.ts:10]: It seems `radiusSegments` should be `radialSegments`.

**Review by Mugen87 — APPROVED** [review]

**Comment by klevron** [comment]
Done, sorry for the multiple commits

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**